src/PlayFabBuddy.Infrastructure/Adapter/PlayFab/PlayerAccountAdapter.cs

```diff
@@ -18,8 +18,8 @@ public PlayerAccountAdapter(PlayFabAdminInstanceAPI adminInstanceApi)
 
     public async Task Delete(MasterPlayerAccountAggregate account)
     {
-        var playedTitleList = await GetPlayedTitleList(account);
-        if (playedTitleList.MoreThanOne())
+        //var playedTitleList = await GetPlayedTitleList(account);
+        if (!account.HasOneTitlePlayerAccount())
```

Comment:
Did you change semantics here on purpose? It was supposed to be `tiles > 1` instead of `titles == 1`

Reply:
Also, this will throw the excepton if tehre is less than one title involved...